src/app/about/page.tsx

```diff
@@ -1,23 +1,50 @@
+        </p>
+        </Section>
+        <p className="mb-4">
+          The Legion of Mary is a lay apostolic association of Catholics who, with the sanction of the Church and under the powerful leadership of Mary Immaculate, Mediatrix of All Graces, serve the Church and their neighbour on a voluntary basis in about 170 countries. The first meeting of the Legion of Mary took place in Myra House, Francis Street, Dublin, Ireland, on 7 September, 1921. This meeting was to have very beneficial consequences for the mission of the Catholic Church and, in a special way, for millions of members of Christ’s lay faithful who would serve in the Legion and for those who would be served by the legionary apostolate. Many persons outside the Catholic Church would also benefit from that apostolate. With the approval and support of the Popes and a great many Bishops, Priests and Religious, as well as the prayers and efforts of legionaries, the Legion, by the grace of God, has grown into a worldwide organisation with several million members.
+        </p>
+        <p className="mb-4">
+          Drawing its inspiration from the True Devotion to Mary, as taught by St. Louis Marie de Montfort, and which had a profound influence on the Founder of the Legion, the Servant of God, Frank Duff, the Legion is at the disposal of the Bishops and Priests for use in the mission of the Church. While essentially a lay association, legionaries look for spiritual and apostolic formation to priests and religious, who, as Legion Spiritual Directors, hold an honoured place in the Legion system. The Legion requires ecclesiastical approval to work in a diocese or parish. Loyalty to the Magisterium and to Ecclesiastical Authority is a basic legionary principle. The Legion aims to bring Mary to the world as the infallible means of winning the world to Jesus and legionary service is based on the doctrine of the Mystical Body of Christ so that in their fellow members and in those they serve, legionaries seek to have the Person of our Lord once again seen and served by Mary, his Mother.
+        </p>
+        <p className="mb-4">
+          The general and essential means by which the Legion of Mary is to effect its object is personal service acting under the influence of the Holy Spirit, having Divine Grace as its moving principle and support, and the glory of God and the salvation of souls as its final end and purpose. Evangelisation, especially the seeking of conversions to the Church, should be a priority for the Legion. Through the visitation of homes and by other means, the Legion must, as a first principle, set out to establish a contact of some sort with every soul everywhere. Seeing and serving Christ in the sick and marginalised is another vital part of the legionary apostolate. While not engaging in the giving of material relief, legionaries will often find opportunities to do works of service for the needy.
```

Comment:
where did you get all these from? Would you mind posting the link and commenting it out somewhere near these quotes? That's what I've started doing anyways, just to keep track of the resources we using.

Reply:
This is all from the og official Legion of Mary website (in ireland). Link for that here: https://www.legionofmary.ie/about. 

I'll put that as a comment in the quotes as well.